src/style/art-gallery.scss

```
@@ -0,0 +1,108 @@
+@import 'var.scss';
+.Art-gallery{
+    padding-top: 70px;
+    background-color: $Black;
```

Давай переменные тоже называть только в camelCase. Чтобы все было единообразно

```suggestion
    background-color: $black;
```